.github/workflows/ci.yml

```diff
@@ -30,6 +30,10 @@ jobs:
           profile: minimal
           toolchain: stable
           override: true
+          components: rustfmt
+      - run: |
+          cargo --version
+          cargo fmt --version
```

Comment:
what's the use of this?

Reply:
Help folks debug the actions failing because there is a mismatch between what they have locally and what is used by the action.